packages/client/src/contents/types/seoMetadata.types.ts

```diff
@@ -32,6 +32,8 @@ export enum SeoSubPageType {
 }
 
 export type GetSEOMetadataQuery = {
+  // The host and protocol of the client (https://hostname.com).
```

Comment:
Add to the message something that alerts the developer to pass the baseUrl besides being optional.